itest/godot/TestRunner.gd

```diff
@@ -10,18 +10,13 @@ func _ready():
 	await get_tree().physics_frame
 
 	var allow_focus := true
-	var unrecognized_args: Array = []
+	var filters: Array = []
 	for arg in OS.get_cmdline_user_args():
 		match arg:
 			"--disallow-focus":
 				allow_focus = false
 			_:
-				unrecognized_args.push_back(arg)
-
-	if unrecognized_args:
-		push_error("Unrecognized arguments: ", unrecognized_args)
-		get_tree().quit(2)
-		return
+				filters.push_back(arg)
```

Comment:
>  Let's at least format it like [a,b,c]

I'm not sure what this means

maybe let's pass --filter or -f in gdscript too before it starts eating argument as filter?

Reply:
I meant to enclose "filter arguments" in a special syntax. Do not make them dependent on argument ordering.

So instead of `a b c` or even `--filter a b c`, you should pass a single command-line argument `"[a,b,c]"`.

Reply:
Okay, I finally understood!